lib/core/config.rb

```diff
@@ -18,6 +18,8 @@ def initialize(args, options)
     load_app_config
     load_apps
 
+    @location = options[:location] || config.dig(:apps, app&.to_sym, :default_location)
```

Comment:
We should use `config.current&.dig(:default_location)` instead. For apps where `match_if_app_name_starts_with` is `true`, `config.dig(:apps, app&.to_sym, :default_location)` wouldn't work, because the `app` provided wouldn't match the app in `controlplane.yml`.

Reply:
Oh! I see
Thanks for the hint.
I will update the code.